Sources/LeafKit/LeafLexer/LeafParameterTypes.swift

```diff
@@ -11,8 +11,8 @@ public enum Parameter: Equatable, CustomStringConvertible {
     case stringLiteral(String)
     case constant(Constant)
     case variable(name: String)
-    case keyword(Keyword)
-    case `operator`(Operator)
+    case keyword(LeafKeyword)
+    case `operator`(LeafOperator)
```

Comment:
Renamed to `Leaf*` as I've gone over those objects enough to be a little more direct about them